### DISCUSSION TITLE
Some contig sequences are not supported well by read mapping in a very low-depth metagenome assembly.

### DISCUSSION THREAD

**Discussion by ilnamkang** [ablab/spades#1257] [Q&A]
### Description of bug

Hi,
I performed a metagenome assembly using the metaSPAdes pipeline. Then, I mapped the input Illumina reads onto "contigs.fasta" file. When I checked the mapping results manually (using IGV), I found some bases are not well supported by read mapping. I have attached a snapshot of IGV that shows a region with many problematic bases.

![spades](https://github.com/ablab/spades/assets/11887252/100e12b9-ab3f-4f24-8987-fa4684c8a3df)

The snapshot is from mapping input reads by bwa mem. But, the pattern is exactly same when I mapped the error-corrected reads or when I used minimap2 (-ax sr) instead of bwa mem.

Please note that the overall sequencing depth was very low, as you may know from the snapshot and log file.

### spades.log

[spades.log](https://github.com/ablab/spades/files/14572413/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/14572419/params.txt)

### SPAdes version

SPAdes v3.15.5

### Operating System

Linux-5.4.0-172-generic-x86_64-with-Ubuntu-18.04-bionic

### Python Version

Python 2.7.17

### Method of SPAdes installation

binaries

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
It is perfectly ok to have short read mappers to struggle aligning reads back to contigs. The key point here is that metaSPAdes emits consensus contigs for the different strains with strain variation, etc. collapsed. As a result, one could have low IDY when reads are aligned back to contigs and therefore read mappers will be unable to align anything.

**Reply by ilnamkang** [reply]
Thank you for the detailed explanation.
